CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,88 @@
+- Add command `antctl upgrade api-storage` to support resource storage version migration for Antrea CRDs. ([#5198](https://github.com/antrea-io/antrea/pull/5198), [@hongliangl])
+- Add support for removing the associated stale conntrack entries when UDP Endpoints are removed, with which UDP requests can be redirected to other Endpoints immediately rather than waiting for the conntrack entries to expire. ([#5112](https://github.com/antrea-io/antrea/pull/5112), [@hongliangl])
+- Add Egress information to flow records for Pod-to-external flows in FlowExporter. ([#5088](https://github.com/antrea-io/antrea/pull/5088), [@dreamtalen])
+- Increase accuracy of Pod information in the flow records by adding a Pod store in FlowExporter and FlowAggregator for them to fetch the Pod information. ([#5185](https://github.com/antrea-io/antrea/pull/5185), [@yuntanghsu])
+- Add support for Service annotation `service.kubernetes.io/topology-mode` in AntreaProxy since the old `service.kubernetes.io/topology-aware-hints` annotation has been deprecated in Kubernetes 1.27. ([#5241](https://github.com/antrea-io/antrea/pull/5241), [@mengdie-song])
+- Support the `service.kubernetes.io/service-proxy-name` label in AntreaProxy to align with KEP 2447. ([#4973](https://github.com/antrea-io/antrea/pull/4973), [@hongliangl])
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Add support for the `sort-by` flag in more `antctl get` commands for more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
+- Add the `kubeAPIServerOverride` option to allow user to override the kube-apiserver address for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
+- Add documentation for deploying Antrea with a Rancher cluster. ([#4733](https://github.com/antrea-io/antrea/pull/4733), [@jainpulkit22])
+
+### Changed
+
+- Multiple APIs are promoted from alpha to beta.
+  - Promote ClusterGroup and Group to v1beta1. ([#5181](https://github.com/antrea-io/antrea/pull/5181), [@GraysonWu])
+  - Promote ExternalIPPool API to v1beta1. ([#5176](https://github.com/antrea-io/antrea/pull/5176), [@hongliangl])
+  - Promote Tier API to v1beta1. ([#5172](https://github.com/antrea-io/antrea/pull/5172), [@GraysonWu])
+  - Promote Egress API to v1beta1. ([#5180](https://github.com/antrea-io/antrea/pull/5180), [@wenqiq])
+  - Promote Upgrade AntreaClusterNetworkPolicy and AntreaNativeNetworkPolicy API to v1beta1. ([#5186](https://github.com/antrea-io/antrea/pull/5186), [@GraysonWu])
+  - Promote Traceflow API to v1beta1. ([#5108](https://github.com/antrea-io/antrea/pull/5108), [@luolanzone])
+  - Add a validation schema for the matchLabels field of the ExternalIPPool CRD. ([#5284](https://github.com/antrea-io/antrea/pull/5284), [@tnqn])
```

Comment:
```suggestion
  - Add a validation schema for the matchLabels field in CRD ExternalIPPool. ([#5284](https://github.com/antrea-io/antrea/pull/5284), [@tnqn])
```

Reply:
I think both works, there is no need to change.

Reply:
Sure, the original one is also okay.